README.md

```diff
@@ -3,6 +3,7 @@
 [![CI status](https://github.com/desbma/shh/actions/workflows/ci.yml/badge.svg)](https://github.com/desbma/shh/actions)
 [![crates.io version](https://img.shields.io/crates/v/systemd-hardening-helper)](https://crates.io/crates/systemd-hardening-helper)
 [![AUR version](https://img.shields.io/aur/version/shh.svg?style=flat)](https://aur.archlinux.org/packages/shh/)
+[![nixpkgs unstable version](https://repology.org/badge/version-for-repo/nix_unstable/shh.svg?header=nixpkgs+unstable)](https://repology.org/project/shh/versions)
```

Comment:
Why not directly link to the `nixos.org` page (https://search.nixos.org/packages?channel=unstable&show=shh&size=1&sort=relevance&type=packages&query=shh) like you did in other link below?

Reply:
will do that